app/views/public_pages/tax_questions.html.erb

```diff
@@ -20,7 +20,7 @@
           <li><%=t("views.public_pages.tax_questions.cannot_answer.refund_amounts") %></li>
         </ul>
 
-        <button class="button button--primary button--wide open-intercom" type="submit">
+        <button class="button button--primary button--wide open-intercom">
```

Comment:
type=submit doesn't do anything for this button since all it does is open the Intercom bubbler rather than submit anything (there's not even a form element).